LFS171x/composer-material/tuna-network/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "engines": {
+    "composer": "^0.19.12"
```

Comment:
Will this work with Composer 0.20, which is what I downloaded today?

Reply:
Yes, but we will need to update the version. I will do this and test it works correctly with Composer 0.20

Reply:
Tested, it works. Will update definitions to use 0.20.0

Reply:
Done